gpkitmodels/GP/aircraft/wing/constant_taper_chord.py

```diff
@@ -5,4 +5,5 @@ def c_bar(lam, N):
     "returns wing chord lengths for constant taper wing"
     eta = np.linspace(0, 1, N)
     c = 2/(1+lam)*(1+(lam-1)*eta)
-    return c, eta
+    cbarmac = 2./3*(1+lam+lam**2)/(1+lam)
+    return c, eta, cbarmac
```

Comment:
do you want to make this a function in parse_results so that wing chord lengths will be in the solution?